src/main/java/org/folio/edge/core/Constants.java

```diff
@@ -26,21 +26,15 @@ private Constants() {
   public static final String SYS_RESPONSE_COMPRESSION = "response_compression";
 
   // System properties for SSL/TLS http server configuration
-  public static final String SYS_HTTP_SERVER_SSL_ENABLED = "http-server.ssl_enabled";
-  public static final String SYS_HTTP_SERVER_KEYSTORE_TYPE = "http-server.keystore_type";
-  public static final String SYS_HTTP_SERVER_KEYSTORE_PROVIDER = "http-server.keystore_provider";
-  public static final String SYS_HTTP_SERVER_KEYSTORE_PATH = "http-server.keystore_path";
-  public static final String SYS_HTTP_SERVER_KEYSTORE_PASSWORD = "http-server.keystore_password";
-  public static final String SYS_HTTP_SERVER_KEY_ALIAS = "http-server.key_alias";
-  public static final String SYS_HTTP_SERVER_KEY_ALIAS_PASSWORD = "http-server.key_alias_password";
-  public static final String SYS_WEB_CLIENT_SSL_ENABLED = "web-client.ssl_enabled";
+  public static final String SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_TYPE = "SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_TYPE";
+  public static final String SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_PATH = "SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_PATH";
```

Comment:
Spring declares spring.ssl.bundle.jks.web-server.keystore.location
Need to check how it can be represented to env variable, probably SPRING_SSL_BUNDLE_JKS_WEB-SERVER_KEYSTORE_LOCATION